prelude/bootstrap.js

```diff
@@ -1702,11 +1703,6 @@ function payloadFileSync(pointer) {
   var ancestor = {};
   ancestor.dlopen = process.dlopen;
 
-  function revertMakingLong(f) {
-    if (/^\\\\\?\\/.test(f)) return f.slice(4);
-    return f;
-  }
-
```

Comment:
This was duplicated

Reply:
It was defined two times, each in a different scope. Now, in dlopen, it's undefined. A single definition should probably be made outside the scope.

Reply:
Fixed, could you try again please?